test/integration/sequelize/query.test.js

```diff
@@ -807,29 +733,33 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
         expect(result[0]).to.deep.equal(expected);
       });
 
-      it('binds named parameters array handles escaped $$', async function () {
+      // this was a legacy band aid that has since been removed, because the underlying issue (transforming bind params in strings) has been fixed.
```

Comment:
Do we need this comment?

Reply:
It's there to explain why a test that does seemingly nothing exists as not transforming the contents of strings seems like the logical thing to do (at least to me) and someone may be tempted to remove it, but I kept the test because we used to have a weird behavior there

Reply:
Ah yes, that's a good reason